src/Httpful/Response.php

```
@@ -794,18 +953,27 @@ private function _interpretHeaders()
-        $this->content_type = $content_type[0] ?? '';
+        $this->content_type = $content_type_parts[0] ?? '';
         if (
-            \count($content_type) === 2
+            \count($content_type_parts) === 2
             &&
-            \strpos($content_type[1], '=') !== false
+            \strpos($content_type_parts[1], '=') !== false
         ) {
             /** @noinspection PhpUnusedLocalVariableInspection */
-            list($nill, $this->charset) = \explode('=', $content_type[1]);
+            list($nill, $this->charset) = \explode('=', $content_type_parts[1]);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$nill'. (undefined)

(UnusedLocalVariable)